### ISSUE TITLE
Display drivetrain status

### ISSUE THREAD

**Issue by kylecorry31** [Gongoliers/Robot2019#9]
Display the encoder values (position and speed) using talon.getPosition() and talon.getVelocity(), and the NavX heading (compass and relative/default).

https://github.com/Gongoliers/Robot2019/blob/bd403e3d7b6a6bab4f14aaf2208fbee49d8fe02a/src/main/java/frc/robot/subsystems/Drivetrain.java#L94

**Comment by kylecorry31** [comment]
The lines in operate should be moved to periodic:

```java
SmartDashboard.putNumber("Encoder Distance", Odometry.getDistance(getLeftDistance(), getRightDistance()));
SmartDashboard.putNumber("Gyro Angle", navX.getAngle());
```

**Comment by NicholasBottone** [comment]
Implemented in #a23b0ad
